datalad/support/third/loris_token_generator.py

```diff
@@ -0,0 +1,31 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### #
+import sys
+import json
+
+if sys.version_info[0] == 2:
+    import urllib2 as urllib_request
+else:
+    from urllib import request as urllib_request
+
+class LORISTokenGenerator(object):
```

Comment:
A short docstring describing its purpose would be most welcomed ;-)

Reply:
done